### PR TITLE
project settings and runtime system

### PR DESCRIPTION
**Pull request by giuliojiang**
Infrastructure java project now uses Maven

changed way to run commands from infrastructure java program

### CODE REVIEW

File: infrastructure/infrastructure/pom.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+						<descriptorRef>jar-with-dependencies</descriptorRef>
+					</descriptorRefs>
+				</configuration>
+			</plugin>
+		</plugins>
+	</build>
+
+	<dependencies>
+		<dependency>
+			<groupId>junit</groupId>
```
Comment (helicopter88) [inline infrastructure/infrastructure/pom.xml]: Latest jUnit is 4.12

File: infrastructure/infrastructure/pom.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	</properties>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.1</version>
+				<configuration>
+					<source>${java.version}</source>
```
Comment (helicopter88) [inline infrastructure/infrastructure/pom.xml]: Ensure that this one is set to 1.8 or 1.7

File: infrastructure/infrastructure/src/main/java/jobmanager/SingleJobConfig.java [hunk 3]
```diff
@@ -57,18 +61,35 @@ public void execute() {
 		// Execute
 		// service_location inputfile outputlocation
 		String command = the_service.getLocation() + " " + input_json_path + " " + output_json_path;
-		Runtime rt = Runtime.getRuntime();
 		System.out.println(command);
+
+		ProcessBuilder pb = new ProcessBuilder(Arrays.asList("/bin/sh", "-c", command));
```
Comment (helicopter88) [inline infrastructure/infrastructure/src/main/java/jobmanager/SingleJobConfig.java]: This looks much better than what you had before
GJ GJ

File: infrastructure/infrastructure/pom.xml (hunk 2 above)
Comment (giuliojiang) [inline infrastructure/infrastructure/pom.xml]: yes, it is on 1.8. The variable is defined just a few lines above that ;)

**Pull request merged**